bluebottle/time_based/migrations/0075_auto_20230127_1552.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.24 on 2023-01-27 14:52
+import datetime
+
+from bluebottle.clients.utils import LocalTenant
+
+from bluebottle.clients.models import Client
+from django.db import migrations, connection
+from django.db.models import F
+
+
+def migrate_back_dated_contributions(apps, schema_editor):
+
+    tenant = Client.objects.get(schema_name=connection.tenant.schema_name)
+    with LocalTenant(tenant):
+        TimeContribution = apps.get_model('time_based', 'TimeContribution')
+
+        # contribution_type = 'period'
+        # c.start < c.created
+        contributions = TimeContribution.objects.filter(contribution_type='period', start__date__lt=F('created__date'))
+
+        for c in contributions:
```

Comment:
@eodolphi Maybe we can walk ttrough this bit together to see if this correctly covers all cases.